examples/federated-learning/tff/nvflare/docker-image/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#checkov:skip=CKV_DOCKER_2: No need for HEALTHCHECK
```

Comment:
Nit: can you rename this directory from `docker-images` to `container-images`? So we don't depend on a product name.

Reply:
Done

Reply:
I don't see this change :(